src/diff.jl

```diff
@@ -91,7 +93,15 @@ function expand_derivatives(O::Symbolic, simplify=false; occurances=nothing)
         if !istree(arg)
             return D(arg) # Cannot expand
         elseif isa(operation(arg), Sym)
-            return D(arg) # Cannot expand
+            inner_args = arguments(arg)
+            if any(isequal(D.x), inner_args)
+                return D(arg) # base case if any argument is directly equal to the i.v.
```

Comment:
Is this right? For expressions like `d/dt f(t, x(t))`, do we just return `d/dt f(t, x(t))` unexpanded?